### DISCUSSION TITLE
Outside temperature capture

### DISCUSSION THREAD

**Discussion by OceanOv** [Rem0o/FanControl.Releases#400] [General]
Hi! How do I log the outside temperature correctly using a .sensor file? How to use sensors and send information to a file? I have a water cooled system and I need to use tenths of Celsius to support 40c

**Comment by Rem0o** [comment]
Sensor files are just plain text. They are a way for any third party to "import" sensor data into fan control easily.

Simply write

![image](https://user-images.githubusercontent.com/14118956/142949517-e015a88a-935a-4d18-9f91-eddf9f21bf65.png)

to get 45 in FanControl, simple as that. You can use any programing language or script to do so.

**Comment by sl4ppy** [comment]
What is the polling rate of the defined file by FanControl?  It seems to read the initial value fine, but then never updates.

**Reply by Rem0o** [reply]
1Hz. 
Don't forget you need to "save" the file with every change.

**Reply by sl4ppy** [reply]
Thanks for the fast reply!  Ya, using the "with" context in python which should open AND close the file with every write...   I'll keep monkeying with it.
